CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/Heart_Snapshot.json

```diff
@@ -1,8 +1,8 @@
 {
-    "identifier": "Heart Rate Recovery",
+    "identifier": "HeartSnapshot",
     "usesTrackedData": true,
     "shouldHideActions": ["goBackward", "skip"],
-    "progressMarkers": ["sex", "birthYear"],
+    "progressMarkers": [],
```

Comment:
Why do they want to remove the progress markers for the demographics questions? That was something that Design felt strongly should be there.

Reply:
That was something I did, I thought it was bizarre personally, because it says "we will ask you for your sex and age" on the screen right before that so why insert an out of context progress bar?  Was it something they felt strongly about in the vacuum of design or was it something they did user testing for and users actually understood what it was conveying and benefited from it?